test/RevenuePool-test.js

```diff
@@ -302,7 +303,51 @@ contract("RevenuePool", (accounts) => {
     });
   });
 
+  describe("setup merchant manager", () => {
+    it("should set trusted merchant registrars", async () => {
+      await merchantManager.setup(
+        actionDispatcher.address,
+        gnosisSafeMasterCopy.address,
+        proxyFactory.address,
+        [relayer],
+        versionManager.address
+      );
+      expect(await merchantManager.getMerchantRegistrars()).to.have.members([
+        relayer,
+      ]);
+    });
+  });
+
+  describe("remove merchant manager", () => {
+    it("should removed trusted merchant registrars", async () => {
```

Comment:
This sounds better: 

"removing merchant registrars"
"can remove merchant registrars"

Reply:
Updated